cameramodels/pinhole_camera.py

```diff
@@ -487,8 +487,8 @@ def target_size(self, target_size):
             raise ValueError('target_size length should be 2')
         roi_height = self.roi[2] - self.roi[0]
         roi_width = self.roi[3] - self.roi[1]
-        self._binning_x = roi_width / target_size[1]
-        self._binning_y = roi_height / target_size[0]
+        self._binning_x = roi_width / target_size[0]
+        self._binning_y = roi_height / target_size[1]
```

Comment:
Thank you for your modification. Could you add the test for `target_size`?

Reply:
I added target size test and update binning test.